### PR TITLE
fuck: add yes flag command

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Never thought I'd swear on GitHub

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

File: pages/common/fuck.md [hunk 1]
```diff
@@ -10,3 +10,7 @@
 - Try to match a rule for the previous command:
 
 `fuck`
+
+- Try to match a rule for the previous command and confirm the first choice immediately:
```
Comment (marchersimon) [inline pages/common/fuck.md:14]:
```suggestion
- Confirm the first choice immediately (correct argument only depends on level of frustration):
```
It's not immediately obvious what's the difference of the three options.

Comment (CleanMachine1) [reply pages/common/fuck.md:14]: They all do the same thing. I wish it was ` fuck --fuck` would be better.

its just a -y flag, infact -y is a flag you can use.

Comment (CleanMachine1) [reply pages/common/fuck.md:14]:
```suggestion
- Confirm the first choice immediately (correct argument depends on level of annoyance):
```

**Review by Waples — APPROVED** [review]

File: pages/common/fuck.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/fuck.md:14]: I'm fine with both

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**